pdfminer/pdffont.py

```diff
@@ -541,6 +543,13 @@ def get_width(self):
             w = -self.default_width
         return w * self.hscale
 
+    def get_bbox(self, cid: int) -> Optional[Tuple[float, float, float, float]]:
```

Comment:
Is the only case this returns None when

self.glyph_bounding_box isn't none and cid isn't in glyph_bounding_box? If so might be nice to explicitly say so